app/models/apple_credential.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class AppleCredential < ActiveRecord::Base
+  belongs_to :public_feed, class_name: "Feed"
+  belongs_to :private_feed, class_name: "Feed"
+
+  validates_presence_of :public_feed
+  validates_presence_of :private_feed
+  validates_associated :public_feed
+  validates_associated :private_feed
+  validates_presence_of :apple_key_id
+  validates_presence_of :apple_key_pem_b64
```

Comment:
Okay, so i think the remaining TODOs from the slack thread are:

- [ ] Also validate presence of the new `apple_provider_id` field (+test)
- [ ] Add a [conditional validation](https://guides.rubyonrails.org/active_record_validations.html#conditional-validation) method on the 3 `apple_***` fields here
    - Called `def has_any_key_field?` or something
    - Check if any of the 3 `apple_***` fields are `.present?`
    - The result should be - if any `apple_***` field is present, all 3 must be present.  If all 3 are blank, that's also valid.  (+test!)